### DISCUSSION TITLE
How to get Artic code vault bagde

### DISCUSSION THREAD

**Discussion by akbar-ardiansyah** [community/community#18299] [Other Feature Feedback, Questions, & Ideas]
Why didn’t I get the Arctic Code Vault Contributor Badge?

**Comment by schneedotdev** [comment]
The badge was given out back in 2020

**Reply by akbar-ardiansyah** [reply]
How about the YOLO i dint get that

**Reply by PSJ4IL** [reply]
galcy brains

**Comment by Sulagna-Dutta-Roy** [comment]
The Artic Code Vault badge no longer existed now.
You will get the Galaxy Brain badge after answering two discussions successfully.

Kindly checkout this: https://github.com/Schweinepriester/github-profile-achievements

**Reply by Mayank-2545** [reply]
ok will 😷

**Reply by mdnurulmomen** [reply]
If I'm not wrong, According to [Achievements](https://github.com/drknzz/GitHub-Achievements) The YOLO badge is given if someone merged a pull request without a review.

**Reply by anxumalo** [reply]
@mdnurulmomen That's correct!

- You get the **YOLO** badge when you merge a pull request without a review.
- You get the **Galaxy Brain** badge after answering two discussions _successfully_.

A _successful_ response is when the author of the post identifies the response that helped them the most by *marking it as the answer*. This adds the **Marked as answer** label next to that post and creates a snippet of the reply near the top of the post, with a link to the full reply. You may have seen other posts marked as **Answered** ✅ around the Community (see [example](https://github.com/orgs/community/discussions/57484)). This helps in two ways:
  - Allows other GitHub Community members with similar questions to easily find the right answer. ✅
  - Shows appreciation to the Community member with the reply that helped you the most. 🙏🏼

**Reply by misaraadel** [reply]
You get the YOLO badge when you merge a pull request without a review.

**Reply by github-staff** [reply]
Thanks so much for posting in our Community @Sulagna-Dutta-Roy. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by PSJ4IL** [reply]
que bad

**Reply by 4po** [reply]
que bad?

**Comment by Jino0910** [comment]
you can't get it out of time

Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)

https://github.com/Schweinepriester/github-profile-achievements

**Reply by 4po** [reply]
why mine got deleted?

**Comment by dqanshi** [comment]
Just leave ther are more archivements

**Comment by RoshdiRaed** [comment]
Why are you so interested in it?

**Comment by RoshdiRaed** [comment]
Just enjoy the features of this amazing environment

**Comment by edrisranjbar** [comment]
The idea of archiving open source projects is really cool.

**Comment by Simon-Busch** [comment]
Can't get it anymore afaik

**Reply by edrisranjbar** [reply]
Yes you are right; I think they do not give this badge any more.

**Comment by Halle-Group** [comment]
Then which badges are available? How many badges are there?

**Reply by anxumalo** [reply]
Hi @Ono-Saito! 

You can see a list of achievable badges in https://github.com/Schweinepriester/github-profile-achievements, as @Sulagna-Dutta-Roy and @Jino0910 mentioned.

The **seven earnable badges** are listed below - see the above link for details about how to earn them:

1. Pair Extraordinaire
2. Quickdraw
3. Starstruck
4. Galaxy Brain
5. Pull Shark
6. YOLO
7. Public Sponsor

Even though the achievements page lists 11 badges, two of them are no longer earnable (❌) and two others are identified as "Being tested" (so it's still unclear how you earn these ⁉️ ). I wonder if anyone has seen these on a user profile?:

- ❌ Arctic Code Vault Contributor 
- ❌ Mars 2020 Contributor
- ⁉️ Heart On Your Sleeve
- ⁉️ Open Sourcerer

ℹ️  The public beta announcement dates back to June 2022: https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/.

**Comment by anxumalo** [comment]
> Why didn’t I get the Arctic Code Vault Contributor Badge?

@akbar-ardiansyah Congratulations on getting the Pull Shark, Quickdraw, and YOLO badges 🎉 

It looks like the previous replies helped to answer your questions - nice work! 😄 
To show your appreciation for the response that helped you the most, please **mark it as the answer** so that other GitHub community members can easily find it. Thanks for posting

**Comment by talha4real** [comment]
The Artic Code Vault badge no longer existed now.

**Comment by misaraadel** [comment]
You get the YOLO badge when you merge a pull request without a review.

**Comment by raj007-star** [comment]
> Why didn’t I get the Arctic Code Vault Contributor Badge?

It was back in 2020 when the Badge existed.

**Reply by ChandanPolai** [reply]
♥️

**Comment by akashdip2001** [comment]
# It's not possible now in 2024, because it only available in 2020 after 4 years ago.

# try to collect others [batches](https://github.com/Schweinepriester/github-profile-achievements/blob/main/README.md)

<br><br>

## Achievements

| Title | Badge | Earnable? | Earned by |
| --- | --- | --- | --- |
Heart On Your Sleeve | ![Heart On Your Sleeve Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/heart-on-your-sleeve-default.png) | 🔜 (Being tested) | React to something on GitHub with a ❤️ emoji 
Open Sourcerer | ![Open Sourcerer Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/open-sourcerer-default.png) | 🔜 (Being tested) | User had PRs merged in multiple public repositories 
||| 
Pair Extraordinaire | ![Pair Extraordinaire Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/pair-extraordinaire-default.png) | ✔️ | [Coauthored](https://docs.github.com/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) in a merged pull request
Quickdraw | ![Quickdraw Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/quickdraw-default.png) | ✔️ | Closed an issue or a pull request within 5 min of opening
Starstruck | ![Starstruck Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/starstruck-default.png) | ✔️ | Created a repository that has 16 stars
||| 
Galaxy Brain | ![Galaxy Brain Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/galaxy-brain-default.png) | ✔️ | 2 accepted answers
Pull Shark | ![Pull Shark Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/pull-shark-default.png) | ✔️ | 2 pull requests merged
YOLO | ![YOLO Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/yolo-default.png) | ✔️ | Merged own pull request without code review
||| 
Arctic Code Vault Contributor | ![Arctic Code Vault Contributor Achievement Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/arctic-code-vault-contributor-default.png) | ❌ | Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)
Public Sponsor | ![GitHub Sponsor Achievement Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/public-sponsor-default.png) | ✔️ | Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)
Mars 2020 Contributor | ![Mars 2020 Helicopter Contributor Achievement Badge](https://raw.githubusercontent.com/Schweinepriester/github-profile-achievements/main/images/mars-2020-contributor-default.png) | ❌ | Contributed code to repositories used in the [Mars 2020 Helicopter Mission](https://github.com/readme/nasa-ingenuity-helicopter)
<img src="https://user-images.githubusercontent.com/73097560/115834477-dbab4500-a447-11eb-908a-139a6edaec5c.gif">
<br><br>
 
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |
<img src="https://user-images.githubusercontent.com/73097560/115834477-dbab4500-a447-11eb-908a-139a6edaec5c.gif">

**Comment by ppseprus** [comment]
I see this discussion is still open. I believe I have couple of repositories archived at the Arctic Code Vault and I think I used to have the badge on my profile, but now I dont. Who can I ask 1) whether I actually have my repos archived and 2) why my badge disappeared? 🤔

**Comment by ppseprus** [comment]
Is anyone still watching this? 🤔

**Comment by steroSterka** [comment]
**How do I get the Galaxy Brain badge?** 
do the steps:
* Answer at least 2 discussions.
* Note that you cannot mark your answer as the answer to your own question.
* Answer questions asked by users in discussions and get marked.
* For example, if you mark my answer as "Mark as Answer, " I'll get a point for Galaxy Brain.
* Similarly, you will get a point when someone marks your answer as the answer in some discussion.

**Reply by ppseprus** [reply]
Have you actually read what [my question](https://github.com/orgs/community/discussions/18299#discussioncomment-8213622) was? 🤔 

> I see this discussion is still open. I believe I have couple of repositories archived at the Arctic Code Vault and I think I used to have the badge on my profile, but now I dont. Who can I ask 1) whether I actually have my repos archived and 2) why my badge disappeared? 🤔

**Reply by github-staff** [reply]
Thanks so much for posting in our Community @steroSterka. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by 4po** [comment]
why mine got deleted?

**Comment by LabyKreative** [comment]
To earn the Galaxy Brain badge, actively participate by answering at least two discussions. Remember, you can't mark your answers as correct, so focus on providing helpful responses to others. When your answers are marked as correct by other users, you'll earn points towards the badge.

**Comment by Cuervo279** [comment]
> Why didn’t I get the Arctic Code Vault Contributor Badge?

The Artic Code Vault badge no longer existed now. The badge was given out back in 2020
